docker/sirius-firm-deputy-hub/Dockerfile

```diff
@@ -11,6 +11,30 @@ RUN apk update \
     git \
     && update-ca-certificates
 
+### Development with hot reload and debugger
+FROM base AS dev
+WORKDIR /app
+
+RUN go get -u github.com/cosmtrek/air && go install github.com/go-delve/delve/cmd/dlv@latest
+EXPOSE 8080
+EXPOSE 2345
+
+ENTRYPOINT ["air"]
+
+FROM node:14.17.5-alpine3.14 as asset-env
```

Comment:
Are these the latest images? We should make sure we are starting this project on the freshest footing rather than just copying the existing setup.

Reply:
latest images for both node and golang added to both docker files